api/app/controllers/application_controller.rb

```diff
@@ -1,42 +1,39 @@
 class ApplicationController < ActionController::API
+  # jwt authentication
 
-    # jwt authentication
+  def encode_data(payload)
+    JWT.encode(payload, "my_secret", "HS256")
```

Comment:
Provide your JWT secret as an environment variable as opposed to adding it as plain text in the code

Reply:
Sure, Thankyou